### PR TITLE
mask image to mask image which is at close range

### PR DESCRIPTION
**Pull request by h-kamada**
input mask image, depth
output mask image which is at close range

### CODE REVIEW

File: jsk_pcl_ros/include/jsk_pcl_ros/mask_image_to_depth_considered_mask_image.h [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    virtual void considerdepth
+    (
+     const sensor_msgs::PointCloud2::ConstPtr& point_cloud2_msg,
+     const sensor_msgs::Image::ConstPtr& image_msg);
+  
+    ////////////////////////////////////////////////////////
+    // ROS variables
+    ////////////////////////////////////////////////////////
+    bool approximate_sync_;
+    int extract_num;
```
Comment (garaemon) [inline jsk_pcl_ros/include/jsk_pcl_ros/mask_image_to_depth_considered_mask_image.h]: extract_num_

Comment (garaemon) [inline jsk_pcl_ros/include/jsk_pcl_ros/mask_image_to_depth_considered_mask_image.h]: _ suffix is needed to obey jsk_pcl_ros convention

File: jsk_pcl_ros/src/mask_image_to_depth_considered_mask_image_nodelet.cpp [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    sub_input_.unsubscribe();
+    sub_image_.unsubscribe();
+  }
+
+  void MaskImageToDepthConsideredMaskImage::considerdepth
+  (
+     const sensor_msgs::PointCloud2::ConstPtr& point_cloud2_msg,
+     const sensor_msgs::Image::ConstPtr& image_msg)
+  {
+    ROS_INFO("a");
```
Comment (garaemon) [inline jsk_pcl_ros/src/mask_image_to_depth_considered_mask_image_nodelet.cpp]: Remove debug messages

**Comment by garaemon** [comment]
Add documentation to README.md

**Comment by h-kamada** [comment]
Fixed. Added dynamic reconfigure config.

**Comment by garaemon** [comment]
Please use the same name to nodelet class as dynamic reconfigure config

**Comment by h-kamada** [comment]
changed name and modified README

**Pull request merged**